src/app/(dashboard)/dashboard/FormList.tsx

```diff
@@ -40,36 +46,62 @@ function FormCard({ form }: { form: IForm }) {
                 </div>
                 <div className="self-end">
                     {form.status === Status.draft && (
-                        <Button className="text-xs font-normal h-6 text-black">
-                            Edit
-                        </Button>
+                         <Link href={'/builder/' + form._id}>
+                            <Button className="text-xs font-normal h-6 text-black">
+                                Edit
+                            </Button>
+                        </Link>
                     )}
                     {form.status === Status.active && (
```

Comment:
since we are rendering the same item in both active and ended status, can we make use of '||' operator to render the content instead of having separate checks for both

Reply:
Yes, I dont know how i missed this. Will correct and merge